juno/assetlist.json

```diff
@@ -206,7 +206,29 @@
         "symbol": "ASVT",
         "logo_URIs": {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/asvt.png"
-        }
+        },
+           {
+        "description": "DAO dedicated to building tools on the Juno Network",
+        "type_asset": "cw20",
+        "address": "juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+        "denom_units": [
+          {
+            "denom": "cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+            "exponent": 0
+          },
+          {
+            "denom": "joe",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno1n7n7d5088qlzlj37e9mgmkhx6dfgtvt02hqxq66lcap4dxnzdhwqfmgng3",
+        "name": "Joe",
+        "display": "joe",
+        "symbol": "JOE",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/joe.png"
+        },
+        "coingecko_id": "joe"
```

Comment:
From my investigation, this coingeckoId is already taken by TraderJoe. If JoeDAO doesn't yet have a coingeckoID, please remove for now.